cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -1239,14 +1239,14 @@ public void testDurableLogInspectCommandWithValue() throws Exception {
         command.readDurableDataLogWithCustomCallback((op, entry) -> originalOperations.add(DurableLogInspectCommand.getActualOperation(op)),
                 0, wrapper.asReadOnly());
 
-        Map<Long, Long> origOperationsCountMap = getOperationsCountMapByAttributes(originalOperations);
+        Map<Long, Long> origOperationsCountMap = getOperationsCountMapBySequenceNumber(originalOperations);
         Mockito.doReturn(false)
                 .when(command).confirmContinue();
         Mockito.doReturn(1L).when(command).getLongUserInput(Mockito.any());
-        Mockito.doReturn("Attributes").doReturn("value").doReturn("no")
+        Mockito.doReturn("SequenceNumber").doReturn("value").doReturn("no")
```

Comment:
Not sure whether this is intentional.

Reply:
This is intentional. Now number of operations in operation queue will be more as we are maintaining index segment now. Also we found #7166 while getting operation map by attributes.